app/invocation/invocation_action_card.tsx

```diff
@@ -1035,7 +1101,18 @@ export default class InvocationActionCardComponent extends React.Component<Props
                     </div>
                   </div>
                   <div className="action-line">
-                    <div className="action-title">Command details</div>
+                    <div className="action-header">
+                      <div className="action-title">Command details</div>
+                      {this.state.command && (
+                        <OutlinedButton
+                          className="copy-bb-execute-button"
+                          title="Copy as bb-execute"
```

Comment:
If we were showing only an icon here (e.g. a "copy" icon) then the `title` would be useful to clarify what is being copied (it would be displayed as a tooltip on hover).

In this case, the `title` is not needed, since it's identical to the button's text content.

```suggestion
```

Reply:
That makes sense. Removed title